data/scrapper/course_rooms.py

```diff
@@ -60,6 +60,30 @@ def default(self, o):
             return o.__dict__
         raise TypeError("Object of type '{type} is not JSON Serializable".format(o.__class__.__name__))
 
+
+# Used to clean up malformed rooms.
+#
+# The first element is the things to match and the second is the replacement
+#
+# Some examples:
+# [['In Deptt', '0'], 'In Dept']
+# ['In Deptt', 'In Dept']
+MALFORMED_ROOM_FIXES = [
+    [['In Deptt', '0'], 'In Dept'],
+]
+
+
+# This function fixes up some malformed data on IIT Kharagpur's website
+def clean_room(room):
+    for fix in MALFORMED_ROOM_FIXES:
```

Comment:
The logic can be simplified while using a dictionary.

```python
def clean_room(room):
    for k, v in MALFORMED_ROOMS:
        if k == room:
            return v
```

Reply:
wouldn't this not work because the first value is an array?

Reply:
We should make the `MALFORMED_ROOMS` global variable a dictionary such as:
```python
MALFORMED_ROOMS = { 'In Deptt': 'In Dept', '0': 'In Dept' } 
```